### ISSUE TITLE
Make a Font Preview Page?

### ISSUE THREAD

**Issue by koesherbacon** [quodroc/HumbleNewTabPage#32]
I think it would be helpful if you made a Font Preview Page.  Since there's so many to choose from, it would be very helpful to see what they all look like at a glance.  Moreover, it would be especially helpful if they could be sorted by type (serif, sans, handwritten, calligraphy, and so on).

I wouldn't mind making it in the fork I've started, but I cannot seem to find where the fonts are stored and/or pulled from in your repo.

Thanks a lot!

~ Ev

**Comment by quodroc** [comment]
Good idea. Right now it is listing all of your system fonts using [chrome.fontSettings](https://developer.chrome.com/extensions/fontSettings), so you could use the font browser in your OS to preview them. I'm thinking about adding web fonts eventually using Google Fonts or something similar.
